tools/generator/ClassGen.cs

```diff
@@ -20,8 +20,8 @@ public class ManagedClassGen : ClassGen {
 		TypeDefinition t;
 		TypeReference nominal_base_type;
 		
-		public ManagedClassGen (TypeDefinition t)
-			: base (new ManagedGenBaseSupport (t))
+		public ManagedClassGen (CodeGenerationOptions opt, TypeDefinition t)
```

Comment:
Is there a way to avoid this change? In our hoped-for "future POCO world order", isn't `ManagedClassGen` going to become a `<class/>` representation of some form? Meaning we want this to be as simple as possible?

Reply:
Looking into this one, got the `TestField` one figured out